govcd/openapi.go

```diff
@@ -291,7 +291,7 @@ func (client *Client) OpenApiPostItem(apiVersion string, urlRef *url.URL, params
 		}
 
 		// Synchronous task - new item body is returned in response of HTTP POST request
-	case http.StatusCreated:
+	case http.StatusCreated, http.StatusOK:
```

Comment:
As I think about it the status 200 is probably because of better compatibility with load balancer probes, which like `http.StatusOK`. 

Comment in line 269 needs an update (https://github.com/vmware/go-vcloud-director/pull/447/files#diff-585772f0a826713495acbb0951db5ff9d9f252426fca85e2e112581e5d90a29dR269)

Reply:
Comment updated now showing `200OK `code as well